data_model.md

```diff
@@ -0,0 +1,399 @@
+        in units of 1/C. Typically -0.002 to -0.005 per degree C.
+    -   Tracking type: type of tracking system, i.e. fixed, single axis,
+        two axis
+    -   For fixed tilt systems:
+        -   Surface tilt: tilt from horizontal of a fixed tilt system,
+            degrees
+        -   Surface azimuth: azimuth angle of a fixed tilt system,
+            degrees
+    -   For single-axis tracked systems:
+        -   Axis tilt: tilt from horizontal of the tracker axis, degrees
+        -   Axis azimuth: azimuth angle of the tracker axis, degrees
+        -   Ground coverage ratio: ratio of total width of modules on a tracker to the distance between tracker axes. For example, for trackers each with two modules of 1m width each, and a spacing between tracker axes of 7m, the ground coverage ratio is 0.286(=2/7).
+            system
+        -   Backtrack: True/False indicator of if a tracking system uses
+            backtracking
+        -   Maximum rotation angle: maximum rotation from horizontal of
+            a single axis tracker, degrees
+
+Other parameters may be included in under an “extra parameters” key. The
+extra parameters may be used by forecasters when implementing other PV
+models. The framework does not provide a standard set of extra
+parameters or require a particular format – these are up to the site
+owner.
+
+### Observations
+{: .anchor}
+
+An Observation must be associated with a Site. A Site object must 
+be created before an Observation.
+
+Creation of Observations will have the following required parameters:
+
+-   Name: name of the Observation
+-   Variable: variable name, e.g. power, GHI. Each allowed variable has
+    an associated pre-defined unit.
+-   Value type: The type of the data in the observation. Typically
+    interval mean or instantaneous, but additional types may be defined
+    for events.
+-   Interval label - Indicates if a time labels the beginning or the
+    ending of an interval average, or indicates an instantaneous value,
+    e.g. beginning, ending, instant
+-   Site: the site that this Observation was generated for.
+-   Uncertainty: a measure of the uncertainty of the observation values.
+    The format will be determined later.
+
+An optional text description field can record information such as
+instrument type. Additional parameters may be specified under the “extra
+parameters” key.
+
+Each Observation has data associated with it. The format of this data is found in the Data Format section below.
+
+### Forecasts
+{: .anchor}
+
+Each Forecast must be associated with a Site which contains important
+information such as location. A Site object must be created before a
+Forecast can be associated with it.
+
+Creation of Forecasts will have the following required parameters (see
+[Use Cases](https://solarforecastarbiter.org/usecases/#forecastdef)
+for more detailed explanation of forecast parameters):
+
+-   Name: name of the Forecast
```

Comment:
On this item and in the rest of the document we use a colon. I copied the items below from the use cases with a dash. We should make them consistent at least within this document. I like the way we did it for the use cases with italics and a dash e.g. "*Issue time of day* - The time of day..." 

We didn't include a Name attribute in the use cases, but I guess that's a separate issue.